### DISCUSSION TITLE
Banana Pi BPI-R4 failed to compile, sdcard.img.gz is too big

### DISCUSSION THREAD

**Discussion by mdsdtech** [immortalwrt/immortalwrt#1312] [Q&A]
### Describe the bug

I compiled ImmortalWRT for Banana PI BPI-4 and the compile error due to sdcard.img.gz is too big. In my compiled i included the pakacges that i needed such as Passwall, Luci n etc. The errors like below:

```
make[3] -C target/linux install
    WARNING: Image file /home/jack/dock/IMMO/build_dir/target-aarch64_cortex-a53_musl/linux-mediatek_filogic/tmp/gfriendwrt-by-mdsdtech-v240524-r30453-4de560acc8-mediatek-filogic-bananapi_bpi-r4-sdcard.img.gz is too big: 57016320 > 46137344
    ERROR: target/linux failed to build.
make -r world: build failed. Please re-run make with -j1 V=s or V=sc for a higher verbosity level to see what's going on
make: *** [/home/jack/dock/IMMO/include/toplevel.mk:233: world] Error 1`
```

### ImmortalWrt version

Snapshot

### ImmortalWrt release

Snapshot

### ImmortalWrt target/subtarget

Mediatek/Filogic

### Device

Banana Pi BPI-R4

### Image kind

Self-built image

### Steps to reproduce

1. git clone from https://github.com/immortalwrt/immortalwrt
2. cd into the folder.
3. Run command:
 ./scripts/feeds update -a
./scripts/feeds install -a
4. Run command: make menuconfig.
5. Select target device and select packages.
6. run command:
make -j$(nproc) V=s defconfig download clean world

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by dd-bpir3** [comment]
具体记不住了在调整固件大小第一项取消就好了

**Reply by suyuan168** [reply]
救命  怎么调整的啊

**Reply by mdsdtech** [reply]
in config unselect or uncheck CONFIG_TARGET_ROOTFS_INITRAMFS

**Comment by tutugreen** [comment]
同遇到，会不会是需要改这？

https://github.com/immortalwrt/immortalwrt/blob/1ed559c77cb4cf7991a84b15942fc18d784bf310/target/linux/mediatek/image/filogic.mk#L365-L404

**Reply by 1715173329** [reply]
Y

**Reply by suyuan168** [reply]
> 同遇到，会不会是需要改这？
> 
> https://github.com/immortalwrt/immortalwrt/blob/1ed559c77cb4cf7991a84b15942fc18d784bf310/target/linux/mediatek/image/filogic.mk#L365-L404

怎么解决的？
